apis/r/tests/testthat/test-SOMADataFrame.R

```diff
@@ -820,3 +820,28 @@ test_that("factor levels can grow without overlap", {
     ref <- rbind( tibble::as_tibble(tbl_1), tibble::as_tibble(tbl_2) )
     expect_equal(tbl, ref)
 })
+
+test_that("factor levels cannot extend beyond index limit", {
+    for (tp in c("INT8", "UINT8")) {
+        uri <- tempfile()
```

Comment:
This should probably be`withr::local_tempfile()` for consistency and to ensure automatic cleanup of test files

Reply:
Strong disagree.  I remove these as they littler things in my use (often driven with `r`).

Reply:
R itself has a guarantee to clean up testfile in its tempdir at the end of each session. I love to run tests in independent R sessions (via `r` or `Rscript`) yet this gets in the way and it is irritating:

```sh
edd@rob:/tmp$ ls -1d callr-sev-* devtools_install_* scoped_* soma-dataframe-* summarized-experiment* tiledb-rest-result-storage* | wc -l
174
edd@rob:/tmp$ 
```